src/general/NewGameSettings.cs

```diff
@@ -338,6 +349,14 @@ public void OpenFromDescendScreen(GameProperties currentGame)
         planetSettings.SetWorldSeaLevel((int)settings.WorldOceanicCoverage);
         planetSettings.SetWorldGeologicalActivity((int)settings.GeologicalActivity);
         planetSettings.SetWorldClimateInstability((int)settings.ClimateInstability);
+
+        planetSettings.SetSulfideLevel((int)settings.SulfideLevel);
```

Comment:
Why are these converted to ints here? I think it would be better to consistently use the enum in the settings. (even if just above this that is not done, I think using better patterns in new code is preferable)